components/reusable-components/perk-card/PerkCard.tsx

```diff
@@ -8,10 +9,10 @@ export interface PerkCardsData {
 }
 const PerkCard = ({ perkCardText, perkCardIcon, id }: PerkCardsData) => {
   return (
-    <div className={style.perkCardWrapper}>
-      <div className={style.perkCardInner} key={id}>
+    <div className={style.perkCardWrapper} key={id}>
```

Comment:
@Jasminamih you need to add key when you map over the perks card data and show the PerksCard
(that would be in PerksData.tsx)

Reply:
remove the key={id} from here